### ISSUE TITLE
axiswise normalization does not work if type is not specified

### ISSUE THREAD

**Issue by SvenWarnke** [pyviz/holoviews#3841]
Hi,
If I do:

import pandas as pd
import holoviews as hv

hv.extension('bokeh')

df_1 = pd.DataFrame({'x':[1,2,3], 'y':[1, 2, 3]})
df_2 = pd.DataFrame({'x':[1,2,3], 'y':[10, 20, 30]})

(
    hv.Curve(df_1, kdims=['x'], vdims=['y']) + \
    hv.Curve(df_2, kdims=['x'], vdims=[('y', 'not y')])
).options(axiswise=True)

![image](https://user-images.githubusercontent.com/44029716/61708204-83b04d80-ad4c-11e9-8631-01c466be1f12.png)

It does not apply the axiswise property, although the FAQ https://holoviews.org/FAQ.html says it should:  `your_layout.options(axiswise=True)`. 

If I specify `.options('Curve',  axiswise=True)` it works as intended. 

![image](https://user-images.githubusercontent.com/44029716/61708897-3fbe4800-ad4e-11e9-8841-1f5fe623af26.png)

There are 2 things that I don't like in this behavior:
* I need to specify 'Curve' although I feel like the axiswise property belongs to the Layout and not the Curve
* I have specified an alias for y in the second plot. This has the effect, that y axis are not linked. However they still get normalized together. This behavior does not make much sense, imho, if renaming avoids linking the axis, this should also apply to normalization.

What do you think? Thanks!

**Comment by tylermorganme** [comment]
I just spent a day fighting this so thank you for documenting the solution. I agree that the docs don't accurately explain how to apply axiswise normalization.

**Comment by philippjfr** [comment]
Agreed, the docs on normalization seem to have gotten lost almost entirely when we reorganized the user guide.

**Comment by philippjfr** [comment]
Oh, the FAQ is also definitely wrong, I don't think that ever worked.

**Comment by jbednar** [comment]
D'oh! You and I both reviewed that and didn't catch it.  That *should* work, and makes much more sense than what one actually has to do; I really think we should just make it work!

**Comment by philippjfr** [comment]
Just merged which made `shared_axes` on layout plots consistently supported which does what you want.
